tests/e2e/interactive_mg_runner.py

```diff
@@ -96,6 +96,9 @@ def load_args():
 
 
 def _start_instance(name, args, log_file, queries, use_ssl, procdir):
+    assert not (
+        name in MEMGRAPH_INSTANCES.keys()
+    )  # If this raises, you are trying to start an instance with the same name than one already running.
```

Comment:
This is a bit strange ordering, why not:
```suggestion
    assert name not in MEMGRAPH_INSTANCES.keys()
      # If this raises, you are trying to start an instance with the same name than one already running.
```